apps/storybook-react-native/package.json

```diff
@@ -17,8 +17,6 @@
     "@metamask/design-system-twrnc-preset": "workspace:^",
     "@metamask/design-tokens": "workspace:^",
     "@metamask/utils": "^11.7.0",
-    "@solana/addresses": "^2.0.0",
-    "bitcoin-address-validation": ">=2.0.0",
```

Comment:
These were removed in https://github.com/MetaMask/metamask-design-system/pull/816 but dependencies weren't updated